pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -1742,6 +1764,11 @@ func (wf *Workflow) GetResourceScope() ResourceScope {
 	return ResourceScopeLocal
 }
 
+// GetWorkflowSpec returns the Spec of a workflow.
+func (wf *Workflow) GetSpec() WorkflowSpec {
```

Comment:
Shouldn't this be called
```suggestion
func (wf *Workflow) GetWorkflowSpec() WorkflowSpec {
```
? The comment above is correct but the name is not.

Reply:
Otherwise this won't be counted as an interface implementation